app/views/orgs/rosters/_download_csv_modal.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="remodal text-left" data-remodal-id="download-csv-modal">
+ <button data-remodal-action="close" class="remodal-close"><%= octicon 'x' %></button>
+  <h2 class="remodal-header text-left">Select a Grouping</h2>
+
+  <%= link_to 'Download without group names', roster_path(format: :csv), class: 'btn btn-primary btn-sm' %>
+  <% organization.groupings.map do | grouping | %>
+    <h4>
+      <%= link_to grouping.title, roster_path(grouping: grouping.id, format: :csv), class: 'btn btn-primary btn-sm' %>
```

Comment:
Let's give this a tiny touch up:

![screen shot 2018-07-02 at 1 28 58 pm](https://user-images.githubusercontent.com/11095731/42184963-1583ba2a-7dfc-11e8-905a-f0e4b81b4c98.png)

To get this ☝️ use the following styles:
```erb
  <%= link_to 'Download without group names', roster_path(format: :csv), class: 'btn btn-primary btn-sm btn-block' %>
  <% organization.groupings.map do | grouping | %>
    <h4>
      <br>
      <%= link_to(
          "Download #{grouping.title}",
          roster_path(grouping: grouping.id, format: :csv),
          class: 'btn  btn-sm btn-block'
        )
      %>
    </h4>
  <% end %>
```

Reply:
looks good! thanks!